test/onnx/onnx_test_common.py

```diff
@@ -286,35 +285,19 @@ def run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
         # Feed args and kwargs into exporter.
         # Note that exporter should flatten kwargs into positional args the exported model;
         # since ONNX doesn't represent kwargs.
-        export_error: Optional[torch.onnx.OnnxExporterError] = None
-        try:
-            with _dynamo_config.patch(do_not_emit_runtime_asserts=True):
-                onnx_program = torch.onnx.dynamo_export(
-                    ref_model,
-                    *ref_input_args,
-                    **ref_input_kwargs,
-                    export_options=torch.onnx.ExportOptions(
-                        dynamic_shapes=self.dynamic_shapes,
-                        diagnostic_options=torch.onnx.DiagnosticOptions(
-                            verbosity_level=logging.DEBUG
-                        ),
+        with _dynamo_config.patch(do_not_emit_runtime_asserts=True):
+            onnx_program = torch.onnx.dynamo_export(
+                ref_model,
+                *ref_input_args,
+                **ref_input_kwargs,
+                export_options=torch.onnx.ExportOptions(
+                    dynamic_shapes=self.dynamic_shapes,
+                    diagnostic_options=torch.onnx.DiagnosticOptions(
+                        verbosity_level=logging.DEBUG
                     ),
-                )
-        except torch.onnx.OnnxExporterError as e:
-            export_error = e
-            onnx_program = e.onnx_program
-
-        if diagnostics.is_onnx_diagnostics_log_artifact_enabled():
```

Comment:
onnx_diagnostics still exists

Reply:
the exception no longer stores the onnx program so this logic is no longer relavent. Also this is just dumping the sarif file we don't need.